### ISSUE TITLE
Missing `data/exp_pdb_seq.json`

### ISSUE THREAD

**Issue by wangz10** [zzhangzzhang/pLMs-interpretability#1]
Hi @zzhangzzhang , 

Great study! I'm interested in using the Categorical Jacobian method as a evaluation tool for protein LMs. Can you please provide this `data/exp_pdb_seq.json` file or how to generate this file from a list of PDB IDs?

Curious what's the angstrom cutoffs used for defining contacts between residues. 

Thanks,
Zichen

**Comment by zzhangzzhang** [comment]
Hi Zichen, 

Thanks! Here is the exp_pdb_seq.json file. 
[exp_pdb_seq.json](https://github.com/user-attachments/files/17667319/exp_pdb_seq.json)

or if you want to get sequences from your own pdb file, you can use:

```
d3to1 = {'CYS': 'C', 'ASP': 'D', 'SER': 'S', 'GLN': 'Q', 'LYS': 'K',
'ILE': 'I', 'PRO': 'P', 'THR': 'T', 'PHE': 'F', 'ASN': 'N', 
'GLY': 'G', 'HIS': 'H', 'LEU': 'L', 'ARG': 'R', 'TRP': 'W', 
'ALA': 'A', 'VAL':'V', 'GLU': 'E', 'TYR': 'Y', 'MET': 'M'}

def get_sequence_from_pdb(pdb_file):
    parser = PDBParser(QUIET=True)
    structure = parser.get_structure('PROTEIN', pdb_file)    

    for model in structure:
        for chain in model:
            seq = []
            for residue in chain:
                if residue.resname not in d3to1:
                    return None
                seq.append(d3to1[residue.resname])
            return ''.join(seq)

```

Also, when I was doing this study I used PDB files, that's why I needed to extract sequence + align. However, recently I realised that CIF files are so much better, you can just get the experimental sequence from the 'pdbx_seq_one_letter_code_can' field and the correct residue id from '_atom_site.label_seq_id'.

Regarding the cutoff, we added more detailed description now in the PNAS version. It's CA-CA distance 
 10 Å. Another correction we made during revision is the value we calculated is P@L/2, instead of P@L.

**Comment by wangz10** [comment]
Thank you for the information @zzhangzzhang !  Just curious, why did you change the metric from P@L to P@L/2?

**Comment by zzhangzzhang** [comment]
We did not change the metric. It was a typo in labeling only, we calculated P@L/2 in both the biorxiv and final publication version. 

The reason we decide to use P@L/2 as metric rather than the more commonly used P@L is because we noticed that in many proteins, if you only look at long-range interactions between residues separated by 24 aa or more, the number of contacts in experimental structure is less than L (length of protein).
